app/ui/legacy/src/main/res/values/themes.xml

```diff
@@ -5,12 +5,7 @@
         <item name="android:windowBackground">@android:color/black</item>
     </style>
 
-    <!-- Empty base themes that can be easily replaced by RRO (Runtime Resource Overlay) themes -->
-    <style name="Theme.K9.Light.Base" parent="Theme.Material3.Light.NoActionBar" />
-    <style name="Theme.K9.Dark.Base" parent="Theme.Material3.Dark.NoActionBar" />
-
-    <style name="Theme.K9.Light.Common" parent="Theme.K9.Light.Base">
-        <!-- Colors -->
+    <style name="Theme.K9.Light" parent="Theme2.Main.Light">
```

Comment:
Wouldn't it be easier to inherit from `Theme2.K9.Light` to avoid duplication of `colorPrimary` etc?

Reply:
This whole layer should disapear with the next changes.